File: app/src/main/java/com/nfaralli/particleflow/WPPreferenceFragment.java
package com.nfaralli.particleflow;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;

public class WPPreferenceFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.wallpaper_preference, rootKey);
    }
}

File: app/src/main/kotlin/com/nfaralli/particleflow/WPPreferenceFragment.kt
package com.nfaralli.particleflow

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat

class WPPreferenceFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.wallpaper_preference, rootKey)
    }
}
